src/ChildProcessUtilities.js

```diff
@@ -50,4 +61,23 @@ export default class ChildProcessUtilities {
       childProcess.stderr.setEncoding("utf8").on("data", chunk => stderr += chunk);
     }
   }
+
+  static registerChild(child) {
+    children.push(child);
+    child.on("exit", () => {
+      children.splice(children.indexOf(child), 1);
```

Comment:
Yes.  I initially thought I was going to have to propagate the signal through to children on `CTRL-C`, but it turns out that happens automatically.

Pushed a patch that simplifies this (and rebased).